sdk/eventhubs/azure_messaging_eventhubs/src/common/recoverable/connection.rs

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation. All Rights reserved
 // Licensed under the MIT license.
 
+// cspell: ignore Cloneable
```

Comment:
Did this work? I thought there had to be no spaces e.g.,


```suggestion
// cspell:ignore Cloneable
```

Reply:
It has in the past. And cspell doesn't seem to complain. I'll check.